fontforge/ffpython.h

```diff
@@ -138,7 +139,7 @@ typedef struct ff_contour {
     int spiro_cnt;
     char *name;
 } PyFF_Contour;
-static PyTypeObject PyFF_ContourType;
+extern PyTypeObject PyFF_ContourType;
```

Comment:
static->extern? Same for PyFF_LayerType

Reply:
Wasn't that a static declaration in a header file? Doesn't that lead to one object of that type per object file that includes it?

Offhand I'm not certain about this so much as generally wary of seeing "static" in header files except for very unusual circumstances.

Reply:
Yeah if it was truly static it should probably just be moved into python.c.